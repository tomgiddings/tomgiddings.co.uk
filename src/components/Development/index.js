// @flow
import React from 'react';
import GitHub from '../GitHub';

class Development extends React.Component<*> {
  render() {
    document.title = 'Tom Giddings - Web Developer';
    return (
      <article className="page development">
        <div className="content-wrapper">
          <h1>Web Developer</h1>
          <p>
            I have worked on both front and back end code using React, Node.js, CSS/SCSS, Javascript, Angular, Git, MySQL, MongoDB, Ruby &amp; Rails and ColdFusion.
            I'm also happy configuring and maintaining VMware and Azure environments.
          </p>
          <p>
            I have experience working on everything from small individual projects to maintaining large platforms as part of a development team and I'm always excited to learn new technologies and languages.
          </p>
          <p>
            You can find me on <a href="https://github.com/tomgiddings"><i className="fab fa-github-alt">&nbsp;</i>GitHub</a>
            , and view my work experience on <a href="https://www.linkedin.com/in/tomgiddings/"><i className="fab fa-linkedin">&nbsp;</i>LinkedIn</a> or <a href="https://stackoverflow.com/story/tomgiddings"><i className="fab fa-stack-overflow">&nbsp;</i>Stack Overflow</a>.
          </p>
          <GitHub user="tomgiddings" />
        </div>
      </article>
    );
  }
}
export default Development;
