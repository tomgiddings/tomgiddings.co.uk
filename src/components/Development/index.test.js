import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Development from './';

afterEach(cleanup);

it('Renders correctly', () => {
  const { asFragment } = render(<Development />);
  expect(asFragment()).toMatchSnapshot();
});
