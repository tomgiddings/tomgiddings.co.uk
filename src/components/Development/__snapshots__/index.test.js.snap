// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<DocumentFragment>
  <article
    class="page development"
  >
    <div
      class="content-wrapper"
    >
      <h1>
        Web Developer
      </h1>
      <p>
        I have worked on both front and back end code using React, Node.js, CSS/SCSS, Javascript, Angular, Git, MySQL, MongoDB, Ruby & Rails and ColdFusion. I'm also happy configuring and maintaining VMware and Azure environments.
      </p>
      <p>
        I have experience working on everything from small individual projects to maintaining large platforms as part of a development team and I'm always excited to learn new technologies and languages.
      </p>
      <p>
        You can find me on 
        <a
          href="https://github.com/tomgiddings"
        >
          <i
            class="fab fa-github-alt"
          >
             
          </i>
          GitHub
        </a>
        , and view my work experience on 
        <a
          href="https://www.linkedin.com/in/tomgiddings/"
        >
          <i
            class="fab fa-linkedin"
          >
             
          </i>
          LinkedIn
        </a>
         or 
        <a
          href="https://stackoverflow.com/story/tomgiddings"
        >
          <i
            class="fab fa-stack-overflow"
          >
             
          </i>
          Stack Overflow
        </a>
        .
      </p>
      <h3>
        What I'm currently using most
      </h3>
      <p>
        At the moment I'm making use of Node.js, React and MongoDB for my projects.
      </p>
      <div
        class="gallery"
      >
        <img
          alt="Node.js"
          src="/logos/nodejs.png"
        />
        <img
          alt="React"
          src="/logos/react.png"
        />
        <img
          alt="MongoDB"
          src="/logos/mongo.png"
        />
      </div>
    </div>
  </article>
</DocumentFragment>
`;
